android/src/main/java/com/reactlibrary/JudoReactNativeModule.kt

```diff
@@ -386,239 +403,370 @@ class JudoReactNativeModule internal constructor(context: ReactApplicationContex
-            googlePayConfiguration?.getBoolean("isBillingAddressRequired")
-        } catch (_: Exception) {
-            null
+            if(googlePayConfiguration?.hasKey("isBillingAddressRequired")!!) {
+                googlePayConfiguration?.getBoolean("isBillingAddressRequired")
+            } else {
+                throw(Exception("isBillingAddressRequired code not found in googlePayConfiguration object"))
+            }
+        } catch (error: Exception) {
+            throw (error)
         }
 
     private val ReadableMap.isShippingAddressRequired: Boolean?
         get() = try {
-            googlePayConfiguration?.getBoolean("isShippingAddressRequired")
-        } catch (_: Exception) {
-            null
+            if(googlePayConfiguration?.hasKey("isShippingAddressRequired")!!) {
+                googlePayConfiguration?.getBoolean("isShippingAddressRequired")
+            } else {
+                throw(Exception("isShippingAddressRequired code not found in googlePayConfiguration object"))
+            }
+        } catch (error: Exception) {
+            throw (error)
         }
 
     private val ReadableMap.billingAddressParameters: ReadableMap?
         get() = try {
-            googlePayConfiguration?.getMap("billingAddressParameters")
-        } catch (_: Exception) {
-            null
+            if(googlePayConfiguration?.hasKey("billingAddressParameters")!!) {
+                googlePayConfiguration?.getMap("billingAddressParameters")
+            } else {
+                null
+            }
+        } catch (error: Exception) {
+            throw (error)
         }
 
     private val ReadableMap.shippingAddressParameters: ReadableMap?
         get() = try {
-            googlePayConfiguration?.getMap("shippingAddressParameters")
-        } catch (_: Exception) {
-            null
+            if(googlePayConfiguration?.hasKey("shippingAddressParameters")!!) {
+                googlePayConfiguration?.getMap("shippingAddressParameters")
+            } else {
+                null
+            }
+        } catch (error: Exception) {
+            throw (error)
         }
 
     private val ReadableMap.isBillingPhoneNumberRequired: Boolean?
         get() = try {
-            billingAddressParameters?.getBoolean("isPhoneNumberRequired")
-        } catch (_: Exception) {
-            null
+            if(billingAddressParameters?.hasKey("isPhoneNumberRequired")!!) {
+                billingAddressParameters?.getBoolean("isPhoneNumberRequired")
+            } else {
+                throw(Exception("isPhoneNumberRequired code not found in billingAddressParameters object"))
+            }
+        } catch (error: Exception) {
+            throw (error)
         }
 
     private val ReadableMap.addressFormat: Int?
         get() = try {
-            billingAddressParameters?.getInt("addressFormat")
-        } catch (_: Exception) {
-            null
+            if(billingAddressParameters?.hasKey("addressFormat")!!) {
```

Comment:
@gcojocaru let's make this similar to iOS implementation,
we have in Kotlin extension fuctions https://kotlinlang.org/docs/reference/extensions.html